src/tracee/seccomp.c

```diff
@@ -271,6 +343,12 @@ static int handle_seccomp_event_common(Tracee *tracee)
 		break;
 	}
 
+	case PR_void:
```

Comment:
I really wouldn't want `PR_void` handler which just returns 0. I think that in current form this is likely to introduce hard to debug problems.

In what case it is used?, maybe I can help finding proper solution.

Reply:
Actually look like it is working now...first test only...I was hosing the CPPFLAGS when setting my define.  Using `+=` solved that issue.  I have some more testing to do, but looks like it is close.